biglog/segment.go

```diff
@@ -164,6 +173,9 @@ func loadSegment(indexPath string) (seg *segment, err error) {
 	seg.setNextOffsets()
 	seg.setCreatedTS()
 
+	// TODO: when this is called via createSegment the index is
+	// created by createSegIndex which doesn't write this timestamp?!
```

Comment:
Good catch, looks like is both:

In one hand it’s actually the [timestamp of the first entry](https://github.com/ninibe/netlog/blob/054cdc47e1a7af2e615f65004fb28bee06900755/biglog/segment.go#L82), not the creation of the segment, which is not always the same since netlog [splits in the background](https://github.com/ninibe/netlog/blob/054cdc47e1a7af2e615f65004fb28bee06900755/topic.go#L212).

Then it seems to be only set [on segment load](https://github.com/ninibe/netlog/blob/master/biglog/segment.go#L165), and not when created at runtime.

Pretty sloppy, I’m open to suggestions.